agent/agentexec/main_test.go

```diff
@@ -0,0 +1,27 @@
+package agentexec_test
```

Comment:
Should have linux build tags and be named `main_linux_test.go`

Reply:
Since the other test file that references `TestBin` compiles on all platforms you will also need to move the var there